### PR TITLE
[DO NOT MERGE] groupnorm scale

### PR DESCRIPTION
**Pull request by mjamroz**
I hope that's what you expected

### CODE REVIEW

File: scripts/classification/imagenet/train_imagenet.py [hunk 1]
```diff
@@ -9,102 +16,243 @@
...
-                        help='path of parameters to load from.')
-    parser.add_argument('--resume-states', type=str, default='',
-                        help='path of trainer state to load from.')
-    parser.add_argument('--log-interval', type=int, default=50,
-                        help='Number of batches to wait before logging.')
-    parser.add_argument('--logging-file', type=str, default='train_imagenet.log',
-                        help='name of training log file')
-    parser.add_argument('--use-gn', action='store_true',
-                        help='whether to use group norm.')
+    parser = argparse.ArgumentParser(
```
Comment (zhanghang1989) [inline scripts/classification/imagenet/train_imagenet.py]: I don't think this code style change is necessary. @hetong007

Comment (mjamroz) [reply scripts/classification/imagenet/train_imagenet.py]: @zhanghang1989  sorry for that, but every time im saving python file it runs code formatter (black) which im using in my daily work. Of course it makes no sense to have only one/two file formatted. Anyway I had to edit `train_imagenet` to import back GroupNorm from opencv, not mxnet as i had in previous PR.

Comment (hetong007) [reply scripts/classification/imagenet/train_imagenet.py]: @mjamroz is there a way to pause your code formatter? The style changes are indeed unexpected, and they make review a bit more challenging (too many diffs).

Comment (mjamroz) [reply scripts/classification/imagenet/train_imagenet.py]: ok, next try @hetong007  @zhanghang1989 
https://github.com/dmlc/gluon-cv/pull/1216

**Pull request closed without merging**